suite-common/wallet-core/src/fees/feesReducer.ts

```diff
@@ -40,3 +42,39 @@ export const feesReducer = createReducer(initialState, builder => {
 
 export const selectNetworkFeeInfo = (state: FeesRootState, networkSymbol?: NetworkSymbol) =>
     networkSymbol ? state.wallet.fees[networkSymbol] : null;
+
+export const selectNetworkFeeLevel = (
+    state: FeesRootState,
+    level: FeeLevelLabel,
+    networkSymbol?: NetworkSymbol,
+): FeeLevel | null => {
+    const networkFeeInfo = selectNetworkFeeInfo(state, networkSymbol);
+    if (!networkFeeInfo) return null;
+
+    const feeLevel = networkFeeInfo.levels.find(x => x.label === level);
+
+    return feeLevel ?? null;
+};
+
+export const selectNetworkFeeLevelTimeEstimate = (
+    state: FeesRootState,
+    level: FeeLevelLabel,
+    networkSymbol?: NetworkSymbol,
+): string | null => {
+    const networkFeeInfo = selectNetworkFeeInfo(state, networkSymbol);
+    const feeLevel = selectNetworkFeeLevel(state, level, networkSymbol);
+    if (!feeLevel || !networkFeeInfo) return null;
+
+    return formatDuration(networkFeeInfo.blockTime * feeLevel.blocks * 60);
+};
+
+export const selectNetworkFeeLevelFeePerUnit = (
+    state: FeesRootState,
+    level: FeeLevelLabel,
+    networkSymbol?: NetworkSymbol,
+): string | null => {
+    const feeLevel = selectNetworkFeeLevel(state, level, networkSymbol);
```

Comment:
nit 
```suggestion
 => selectNetworkFeeLevel(state, level, networkSymbol)?.feePerUnit ?? null
```

Reply:
I prefer to not directly access props of a function call. Might lead to unreadable code. So I will resolve this.